### DISCUSSION TITLE
Dashboard Map view and custom pins

### DISCUSSION THREAD

**Discussion by kalib-code** [directus/directus#9945] [Feature Requests]
I would like to request a map view on dashboard its great for map viewing data

**Comment by rijkvanzanten** [comment]
Love that idea. Should be fairly straightforward, as we already have all the mapping infrastructure figured out in the layout / interface. @Oreilles how hard/easy would it be to reuse the map used on the map interface elsewhere?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
